packages/manager/apps/web/client/app/configuration/configuration.constants.js

```diff
@@ -0,0 +1,8 @@
+export const OVH_WEB_OFFERS_2024 =
+  'https://www.ovhcloud.com/fr/lp/webcloud-offers/';
+
+export const OFFERS_BANNER_ALT = 'common_offers_banner_alt';
```

Comment:
What's the need to store the translation key in constant file?

Reply:
Yes not really needed, could be also in the plain html as a static string